pkg/machine/qemu/machine.go

```diff
@@ -215,11 +215,7 @@ func (v *MachineVM) addMountsToVM(opts machine.InitOptions) error {
 		target := extractTargetPath(paths)
 		readonly, securityModel := extractMountOptions(paths)
 		if volumeType == VolumeTypeVirtfs {
-			virtfsOptions := fmt.Sprintf("local,path=%s,mount_tag=%s,security_model=%s", source, tag, securityModel)
-			if readonly {
-				virtfsOptions += ",readonly"
-			}
-			v.CmdLine = append(v.CmdLine, []string{"-virtfs", virtfsOptions}...)
+			v.CmdLine = *v.CmdLine.WithVirtfsMount(source, tag, securityModel, readonly)
```

Comment:
if this uses the builder pattern than I don't think you need to assign the slice every time again,
just writing `v.CmdLine.WithVirtfsMount(source, tag, securityModel, readonly)` should already modify the slice in place

Reply:
correct.